localization/english/event_localization/wc_central_kalimdor_events_l_english.yml

```diff
@@ -290,7 +290,7 @@ cen_kal.1075.desc:0 "I come back to the cave, and find [SCOPE.sC('guide').GetFir
 cen_kal.1075.a:0 "I am in need of a warm drink..."
 cen_kal.1075.b:0 "Maybe this has gone on for too long..."
 
-cen_kal.1076.desc:0 "The drink is nothing like the one from the trial of the Winds. Immediately, my muscles begin to spasm, and I feel nauseous. The pain throughout my body increases and my vision fades to black.\n\nI feel a faint breeze on my face as I regain control of my body. When my sight returns, I look down and see my body has become translucent. The drink must have transported my spirit somewhere else. Investigating the area around me, I notice several large red sandstone formations engraved with the handprints of young and old Taurens who have traveled here. I am currently in the Red Rocks, a burial site sacred to my people. I feel a surge of awe and reverence as I realize that I am standing among the spirits of my ancestors."
+cen_kal.1076.desc:0 "The drink is nothing like the one from the trial of the Winds. Immediately, my muscles begin to spasm, and I feel nauseous. The pain throughout my body increases and my vision fades to black.\n\nI feel a faint breeze on my face as I regain control of my body. When my sight returns, I look down and see my body has become translucent. The drink must have transported my spirit somewhere else. Investigating the area around me, I notice several large red sandstone formations engraved with the handprints of young and old Tauren who have traveled here. I am currently in the Red Rocks, a burial site sacred to my people. I feel a surge of awe and reverence as I realize that I am standing among the spirits of my ancestors."
```

Comment:
the Tauren

Reply:
"handprints of young and old Tauren who have traveled here" sounds pretty grammaticaly correct to me
Is that not the case?